src/.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,28 @@
+name: Run Tests
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
This is what we have in our main `eppo` repository, and tests seem to run every push 🤷‍♂️